src/polyfills/details/_base.scss

```diff
@@ -6,6 +6,10 @@
 summary {
 	// Make sure summary remains visible
 	display: list-item !important;
+	list: {
```

Comment:
@duboisp That's one of the properties this code block will generate.

To prevent list bullets from appearing in browsers that don't recognize ``disclosure-closed``, I had to set two ``list-style`` properties. The first is set to ``none`` to prevent the aforementioned browsers from showing list bullets (since they'll fall back on the first property they understood). The second is set to ``disclosure-closed inside`` to support versions of Firefox that have native details/summary support (and [future versions of Blink](https://bugs.chromium.org/p/chromium/issues/detail?id=590014)).

I felt like writing ``list-style`` twice in a row wouldn't look very clean. So I used [SASS' nested properties extension](http://sass-lang.com/documentation/file.SASS_REFERENCE.html#nested_properties) for this code block.